src/components/templates/protected-area/passwordGenerator.scss

```diff
@@ -0,0 +1,77 @@
+.passwGeneratorContainer {
+  z-index: 99;
+  margin-top: -100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.passwGeneratorModal {
+  min-width: 350px;
+  color: #374049;
+  padding-left: 30px;
+  padding-right: 30px;
+  padding-bottom: 10px;
+  width: 35%;
+  border-radius: 5px;
+  box-shadow: 0px 5px 10px grey;
+}
+
+.modalHeader {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 2px solid #d6d6d6;
+  margin-bottom: 4%;
+}
+
+.closeModal {
+  font-weight: bold;
+}
+.sliderContainer {
+  margin-bottom: 0.8rem;
+}
+
+.slider {
+  -webkit-appearance: none;
+  width: 25%;
+  height: 3px;
+  border-radius: 5px;
+  background: #d1d1d1;
+  outline: none;
+  opacity: 0.7;
+  -webkit-transition: 0.2s;
+  transition: opacity 0.2s;
+}
+
+.slider::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  appearance: none;
+  width: 15px;
+  height: 15px;
+  border-radius: 50%;
+  background: #00477e;
+  cursor: pointer;
+}
+
+.slider::-moz-range-thumb {
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background: #00477e;
+  cursor: pointer;
+}
+
+.modalFooter {
+  margin-top: 15px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.saveModal {
+  min-width: 100px;
+}
```

Comment:
We should wrap all the styling in a single class to avoid spreading in the whole app